src/core/enums/route-names.enum.ts

```diff
@@ -0,0 +1,5 @@
+export enum RouteNames {
```

Comment:
```suggestion
export enum RouteName {
```

enums should be in singular form unless the values are BIT